src/pybind/mgr/dashboard/controllers/home.py

```diff
@@ -21,11 +21,12 @@
 
 class LanguageMixin(object):
     def __init__(self):
+        product_name = os.environ.get('DASHBOARD_BUILD_ASSETS_FROM', '')
         try:
             self.LANGUAGES = {
                 f
                 for f in os.listdir(mgr.get_frontend_path())
-                if os.path.isdir(os.path.join(mgr.get_frontend_path(), f))
+                if os.path.isdir(os.path.join(mgr.get_frontend_path(), product_name, f))
```

Comment:
Why not using ceph options? I'm not familiar with rook but for example, you would have to set the env variable in every mgr host to let these mgr know where to find the dist folder? @nizamial09

Reply:
Nevermind, it is a build process thing XD.